modules/nextflow/src/main/groovy/nextflow/extension/ChannelExtensionDelegate.groovy

```diff
@@ -136,14 +198,23 @@ class ChannelExtensionDelegate implements DelegatingPlugin {
     }
 
     Object invokeExtensionMethod(Object channel, String method, Object[] args) {
-        final target = operatorExtensions.get(method)
+        final target = aliasOperatorExtensions.get(method)?.v2 ?: operatorExtensions.get(method)
+        method = aliasOperatorExtensions.get(method)?.v1 ?: method
         if( target==null )
```

Comment:
What are `v1` and `v2` attributes?

Reply:
ChannelExtensionDelegate maintains a Map<Tuple> where the key is the alias, and the value is a tuple with the real method and the class how implements it so basically:

if we are calling the invokeExtensionMethod with a method not registered as a plugin we call the method as usual
if we are invoking an alias method, we retrieve the extension and the real method from the map

Reply:
ok, think you can copy & paste this as a comment in that code 😄